data/campaigns/Heir_To_The_Throne/scenarios/03_The_Isle_of_Alduin.cfg

```diff
@@ -193,17 +198,22 @@
             speaker=Konrad
             message= _ "So this is Alduin. It looks a little... desolate."
         [/message]
+        [message]
+            speaker="Usadar Q'kai"
+            message= _ "Who is that? Oh, a party of elves has landed. We shall drive them back into the sea!"
+        [/message]
         [message]
             speaker=Delfador
-            message= _ "I fear so, Konrad. It seems that the orcs have come even here. Here to the place where I was born, where I was trained."
+            message= _ "If the orcs have come here, their forces at Elensefar must be even more numerous than I feared."
         [/message]
         [message]
-            speaker="Usadar Q'kai"
-            message= _ "Who is that? Oh, a party of elves has landed. We shall drive them back into the sea!"
+            speaker=Delfador
```

Comment:
So, Delfador has multiple "mood" portraits available; I wonder if it'd make sense to have him use his "mentoring" one here?

Reply:
His default `[unit]profile=` picture has a `~RIGHT()` modifier on it, the one in `DELFADOR_MENTORING_ELF` doesn't. IMO the switch of side breaks the flow of dialogue, so my feeling is against using it.

Reply:
oh ok nvm then

Reply:
It's still there. Assuming you meant to do so, I've marked it as resolved.